node/src/cli.rs

```diff
@@ -184,7 +184,7 @@ pub struct Cli {
     #[arg(long)]
     pub para_id: Option<u32>,
 
-    /// Relay chain arguments, optionally followed by "--" and orchestrator chain arguments
+    /// Relay chain arguments, optionally followed by "--" and container chain arguments
```

Comment:
This comment was wrong, maybe something like this:

```suggestion
    /// Extra arguments, `container-args -- relay-args` or `relay-args` if no `--`
```